src/docfx/config/PreloadConfig.cs

```diff
@@ -31,6 +31,26 @@ internal class PreloadConfig
     /// </summary>
     public PackagePath? FallbackRepository { get; init; }
 
+    /// <summary>
+    /// Specify the publish repository url
+    /// </summary>
+    public string? PublishRepositoryUrl { get; init; }
+
+    /// <summary>
+    /// Specify the publish branch
+    /// </summary>
+    public string? PublishRepositoryBranch { get; init; }
```

Comment:
`DOCFX_PUBLISH_REPOSITORY_BRANCH` is hard to get. We may not need this config for now.